plugins/aea-ledger-fetchai/setup.py

```diff
@@ -35,7 +35,12 @@
     license="Apache-2.0",
     description="Python package wrapping the public and private key cryptography and ledger API of Fetch.AI.",
     packages=find_packages(include=["aea_ledger_fetchai*"]),
-    install_requires=["aea>=1.0.0rc1, <2.0.0", "ecdsa>=0.15", "bech32==1.2.0"],
+    install_requires=[
+        "aea>=1.0.0rc1, <2.0.0",
+        "ecdsa>=0.15",
+        "bech32==1.2.0",
+        "pyaes>=1.6.1",
+    ],
```

Comment:
@panasevychol  thanks for pointing it out!

Reply:
@solarw we should extend the script which checks for dependencies. And probably also have a standalone install in a clean env so we can be sure the plugins are complete.